### DISCUSSION TITLE
Reducing space between system tray icons?

### DISCUSSION THREAD

**Discussion by oroboros** [valinet/ExplorerPatcher#711] [Q&A]
I'm looking for the Goldilocks setting, because large is too big and small is too small! When I choose larger taskbar, there seems to be so much space between system tray icons that if it were reduced, more icons could fit.

<img width="201" alt="image" src="https://user-images.githubusercontent.com/1668665/150626628-a2b635c0-a95e-4099-a425-31a7880f857c.png">
<img width="211" alt="image" src="https://user-images.githubusercontent.com/1668665/150626639-6347111f-6230-470c-9b5d-2eaf25771143.png">

Also, why are wifi-volume-battery grouped together in Win11 like that (no spacing, hover over highlights them as one), and is there a way to show them like that using Win10 view?

**Comment by Anixx** [comment]
This setting is included as "advanced setting" (not accessible via GUI) in 7+TT. But I second this, inclusion the possibility to reduce the space between tray icons in EP would be very useful.

**Reply by oroboros** [reply]
> This setting is included as "advanced setting" (not accessible via GUI) in 7+TT. But I second this, inclusion the possibility to reduce the space between tray icons in EP would be very useful.

I didn't know that - thanks! Oddly enough it doesn't reduce _all_ icons (why are some icons treated differently)?
<img width="163" alt="image" src="https://user-images.githubusercontent.com/1668665/150630662-7c0b643d-5c35-4a79-a704-e4fde2b0b125.png">

**Reply by Anixx** [reply]
It reduces the space between tray icons. Other things are not tray icons but special buttons.

**Reply by oroboros** [reply]
Another thing I didn't know... Odd that they don't share the same behavior. Thanks

**Comment by Anixx** [comment]
> is there a way to show them like that using Win10 view?

When you install EP, you by default is switched to the Win10-like taskbar, so the icons you mentioned are separated.

![изображение](https://user-images.githubusercontent.com/1271736/150629611-4fcc80db-e40f-4c8e-80f3-2b8623f388b8.png)

**Reply by oroboros** [reply]
> When you install EP, you by default is switched to the Win10-like taskbar, so the icons you mentioned are separated.

Not possible to have that Win11 view for those icons, like for some other settings? It owuld be great to also get that control panel popup - the one thing I'd like to have from Win11 to everything else Win10!
